lib/cisco_node_utils/bgp.rb

```diff
@@ -531,6 +527,35 @@ def default_reconnect_interval
       config_get_default('bgp', 'reconnect_interval')
     end
 
+    # Route Distinguisher (Getter/Setter/Default)
```

Comment:
This looks like it's mostly the same code I reviewed for develop. If this is here because of a merge then it would be better if you just committed the merged code first and then submitted the PR for your actual changes.